opensearch_py_ml/ml_models/sentencetransformermodel.py

```diff
@@ -701,6 +701,38 @@ def zip_model(
             )
         print("zip file is saved to " + zip_file_path + "\n")
 
+    def fix_truncation(
```

Comment:
Can we rename this a bit? Maybe `handle_null_truncation` or `fill_null_truncation_field`

Reply:
good call out. And also as this is a private function let's rename it to: `_fill_null_truncation_field`